.circleci/config.yml

```diff
@@ -335,23 +339,45 @@ jobs:
           paths:
             - build
 
-  dist_bintray_nightly:
-    machine: true
+  dist_obs_nightly:
+    docker:
+      - image: straightshoota/osc
```

Comment:
Maybe the Dockerfile should live inside distribution-scripts and we can publish the image under crystallang organization?

Reply:
Yes, that was also my intention. Forgot to mention it.

Reply:
I've pushed it to a new repository at https://github.com/crystal-lang/osc-docker. Having a separate repo might be easier for integration with dockerhub, I think. You can set it up so that the image gets automatically built from the repository.

Reply:
I've just published `crystallang/osc` manually. If we need and automate build we can trigger later from GitHub Actions.